pkg/spdk/engine.go

```diff
@@ -206,7 +207,15 @@ func (e *Engine) Create(spdkClient *spdkclient.Client, replicaAddressMap map[str
 		e.ReplicaAddressMap = replicaAddressMap
 		e.log = e.log.WithField("replicaAddressMap", replicaAddressMap)
 
-		e.checkAndUpdateInfoFromReplicaNoLock()
+		if salvageRequested {
```

Comment:
Does the engine need to try to connect all replicas before a salvage?
If NO, then the mode check inside `updateInfoToFilterReplicaSalvageCandidates` makes no sense.

Reply:
Fixed. Thank you!